src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp

```diff
@@ -0,0 +1,218 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ie_core.hpp>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/blob_utils.hpp"
+#include "functional_test_utils/plugin_cache.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/pass/convert_prc.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "../../shared_tests_instances/skip_tests_check.hpp"
+#include <gtest/gtest.h>
+
+
+namespace ConvLowPrecicionTestNs {
+
+    using namespace ngraph;
+    using namespace ngraph::builder;
+    using namespace ngraph::element;
+    using namespace ngraph::op;
+    using namespace ngraph::opset1;
+    using namespace std;
+
+    using ConvLowPrecisionTestParams = tuple<InferenceEngine::Precision, // Network Precision
+                                             string,                     // Target Device
+                                             map<string, string>,        // Configuration
+                                             Shape,                      // Input Shape
+                                             pair<float, float>,         // FQ Min and Max (before conv)
+                                             std::size_t                 // Levels
+                                             >;
+
+
+class ConvLowPrecisionTest : public testing::WithParamInterface<ConvLowPrecisionTestParams>,
+                            public LayerTestsUtils::LayerTestsCommon {
+    float fqMin = 0.0f;
+    float fqMax = 0.0f;
+    float inputDataResolution = 1.0f;
+
```

Comment:
in few places there are redundant empty lines

Reply:
Fixed